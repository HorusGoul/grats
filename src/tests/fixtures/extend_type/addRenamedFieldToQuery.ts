/** @gqlType */
class Query {
  // No fields
}

/** @gqlField hello */
export function greeting(_: Query): string {
  return "Hello world!";
}
