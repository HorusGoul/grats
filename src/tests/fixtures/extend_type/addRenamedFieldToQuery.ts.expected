-----------------
INPUT
----------------- 
/** @gqlType */
class Query {
  // No fields
}

/** @gqlField hello */
export function greeting(_: Query): string {
  return "Hello world!";
}

-----------------
OUTPUT
-----------------
schema {
  query: Query
}

directive @methodName(name: String!) on FIELD_DEFINITION

directive @exported(filename: String!, functionName: String!) on FIELD_DEFINITION

type Query {
  hello: String @exported(filename: "tests/fixtures/extend_type/addRenamedFieldToQuery.js", functionName: "greeting") @methodName(name: "greeting")
}